docs/topics_5/web-execution-config.adoc

```diff
@@ -3,64 +3,15 @@
 [id='analysis_configuration_{context}']
```

Comment:
ID must match the file name. So change the file name when you have time. NOT NOW